src/cms/views/roles/role_view.py

```diff
@@ -45,10 +44,13 @@ def get(self, request, *args, **kwargs):
         :return: The rendered template response
         :rtype: ~django.template.response.TemplateResponse
         """
-        role_instance = Role.objects.filter(id=kwargs.get("role_id")).first()
-        group_instance = Group.objects.filter(role=role_instance).first()
+        role_instance = (
+            Role.objects.filter(id=kwargs.get("role_id"))
+            .select_related("group")
+            .first()
+        )
         role_form = RoleForm(instance=role_instance)
-        group_form = GroupForm(instance=group_instance)
+        group_form = GroupForm(instance=getattr(role_instance, "group", None))
```

Comment:
Why not just `role_instance.group`? If the group is not yet set, it's `None` anyway, right?

Reply:
Been there, done that ;-) `role_instance` itself can be `None` and then it will throw an error that `None` has no attribute `group`.

Reply:
Ah yeah, sorry :facepalm: